### PR TITLE
Awesome books navigation

### PR DESCRIPTION
**Pull request by NoelLincoln**
Hello 👋

We have implemented the required features in our awesome books project :
- A Navigation bar.
- Three content sections : books list, add book form and contact info.
- Links work as expected

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,30 +1,75 @@
...
+            <li>Our address: Kenya </li>
+          </ul>
+        </section>
+
+      </main>
+
+      <footer class="footer">
+        <p>&copy Copyright</p>
+      </footer>
+    </div>
```
Comment (mwanawabangona) [inline index.html:71]: - [ ] `Required`
- [ ] Amazing work so far! 👏 
- [ ] However, the `copyright` on the contact section isn't put correctly as the wireframe provided 😞 
- [ ] Please see the screenshots below and Kindly correct the error 🥂 

`Your template`
image

`provided wireframe`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @NoelLincoln @MPDADDY ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MPDADDY** [comment]
Implemented changes requested by the previous reviewer.

**Review by usorfaitheloho — APPROVED** [review]
Hi @NoelLincoln  , 
# STATUS: APPROVED! 🥇 💯 

done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**